uPortal-api/uPortal-api-rest/build.gradle

```diff
@@ -14,4 +16,15 @@ dependencies {
 
     compileOnly "${portletApiDependency}"
     compileOnly "${servletApiDependency}"
+
+    testCompile "junit:junit:${junitVersion}"
+    testCompile "org.mockito:mockito-core:${mockitoVersion}"
```

Comment:
junit and mockito are also inherited from parent project.
https://github.com/Jasig/uPortal/blob/7650ffa9bbc01d28691e7bc550d2137acd6ac242/build.gradle#L100-L107